tasks/FindMyMates.tex

```diff
@@ -0,0 +1,52 @@
+\section{Find My Mates [Party Host]}
+
+The robot starts in a room and waits for someone to approach it. The guest knows the name of other guests at the party but doesn’t know how these people look like and where the people are. The robot will gather this information for the guest.
+
+
+\subsection{Main Goal}
+The robot provides the location and unique description of 3 guests of the party as required by another guest.
```

Comment:
This is more clear 
> The robot provides the location [and a comprehensive, unique description] of at least 3 party guests to the operator.

Reply:
I'm alright with your proposed version. But the sentence "at least 3 party guests" makes me wonder if we can just give the bonus points for extra people given their location and unique description (after given the main goal ones, ofc).

Reply:
There might be a fourth if there is enough time for outstanding performance.

Reply:
And the main goal can be more concise like this,

__Main Goal__
The robot reports to the operator the location and description of at least two party guests.

Reply:
We need to be careful with the main goal. If we include the report, points can only be scored if the robot delivers the actual report.